action.yml

```diff
@@ -0,0 +1,21 @@
+name: 'Azure SDK Action'
+description: 'Handle github event'
+inputs:
+  payload:
+    description: 'GitHub event payload'
+    required: true
+  token:
+    description: 'Github event token'
+    required: true
+runs:
+  using: "composite"
+  steps:
+    - run: |
+        echo "Handle webhook event..."
+        cd $GITHUB_ACTION_PATH/src
```

Comment:
Why not just set this as the working directory?

Reply:
I was trying to keep too many files out of the root path, but it's probably premature. I'll re-organize.